pkgs/applications/science/biology/freebayes/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, cmake, gcc, zlib}:
+
+stdenv.mkDerivation rec {
+  name    = "freebayes-${version}";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner  = "ekg";
+    repo   = "freebayes";
+    # rev    = "v${version}";
+    # rev    = "refs/tags/v${version}";
+    rev = "39e5e4bcb801556141f2da36aba1df5c5c60701f";
+    sha256 = "0xb8aicb36w9mfs1gq1x7mcp3p82kl7i61d162hfncqzg2npg8rr";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ cmake gcc zlib ];
+
+  builder = ./builder.sh;
```

Comment:
Is the builder.sh necessary? Could you include all of the build steps inside of the nix expression?

Seems like (in `builder.sh`) lines 9-11 are the `installPhase`, 7 is the default build command, and 5 could be in placed in between the buildPhase and the configurePhase.